addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -73,13 +73,19 @@ class ComposerTextInput extends Component {
         if (!this.composer) {
             return "";
         }
-        if (this.composer.thread && this.composer.thread.model !== 'mail.channel') {
-            if (this.composer.isLog) {
-                return this.env._t("Log an internal note...");
+        if (!this.composer.thread) {
+            return "";
+        }
+        if (this.composer.thread.model === 'mail.channel') {
+            if (this.composer.thread.correspondent) {
+                return _.str.sprintf("Message %s...", this.composer.thread.correspondent.nameOrDisplayName)
```

Comment:
should add `correspondent` in `useStore` to ensure proper refresh in case of name update